protocol-designer/src/pages/Onboarding/WizardBody.tsx

```diff
@@ -190,6 +190,7 @@ export function WizardBody(props: WizardBodyProps): JSX.Element {
                 border-radius: ${BORDERS.borderRadius16};
               `}
               autoPlay
+              muted={true}
```

Comment:
not necessary but slight nit i'd like to point out. since `muted` is a boolean, you can do something like this instead of explicitly defining it as true.

```suggestion
              muted
```